src/app/app.module.ts

```diff
@@ -13,8 +13,11 @@ import { ReactiveFormsModule } from '@angular/forms';
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 import { BsDropdownModule } from 'ngx-bootstrap/dropdown';
 import { AlertModule } from 'ngx-bootstrap/alert';
+import { FormsModule } from '@angular/forms';
 import { DatePipe } from '@angular/common';
-
+import { HttpClientModule } from '@angular/common/http';
+import { TodoServiceService } from './services/todo-service.service';
```

Comment:
why the name TodoServiceService ?

Reply:
The file name was todo service so it add the full name, I'll check if I can remove one service.